### ISSUE TITLE
Global footer navigation should not use five separate `nav` landmarks

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-mu-plugins#402]
The global footer navigation is currently structured as five separate navigation landmarks. This is only one navigation group, and should be in a single navigation menu.

Additionally, these navigation menus have no accessible name. As individual sections, there isn't really any reasonable name for these menus - they aren't distinct collections of information. But as a single navigation menu, 'Footer' would be a perfectly reasonable name for the menu.

This would significantly help in reducing aural clutter for screen readers, since there are currently 9 navigation landmarks in global scope.

**Comment by ryelle** [comment]
Technically, this is because each column of links is [a separate nav block](https://github.com/WordPress/wporg-mu-plugins/blob/514ec641aec9f89096da8be5892cf7b60ebcc856/mu-plugins/blocks/global-header-footer/footer.php#L24). We could investigate using one nav block and using CSS to make them columns, maybe.

There are a few other requirements, which could probably be flexible

- Each column's content should stay consistent across screen sizes, and reading order should be down each column. They are roughly in content-groups, even if it's not very distinct.
- Each link should stay as a `navigation-link` block, since there's a filter applied to update the URLs on the rosetta (locale-specific) sites, like https://ja.wordpress.org/.

There's some additional discussion about the content of these menus here: https://github.com/WordPress/wporg-mu-plugins/issues/364

**Comment by joedolson** [comment]
It would definitely improve the end user experience to have only one nav block with CSS to manipulate the layout. Alternately, having multiple lists inside a single nav container, but I don't know if that's achievable using the nav block.
